sdk/resourcemanager/Azure.ResourceManager/tests/Scenario/ResourceLinkCollectionTests.cs

```diff
@@ -11,90 +11,90 @@
 
 namespace Azure.ResourceManager.Tests
 {
-    public class ResourceLinkCollectionTests : ResourceManagerTestBase
-    {
-        public ResourceLinkCollectionTests(bool isAsync)
-            : base(isAsync)//, RecordedTestMode.Record)
-        {
-        }
+    //public class ResourceLinkCollectionTests : ResourceManagerTestBase
```

Comment:
consider to just remove test cases

Reply:
I would like to keep it in case we could easily find it when we want to put them back